dataplane/src/main.rs

```diff
@@ -19,7 +19,7 @@ use args::{CmdArgs, Parser};
 #[global_allocator]
 static GLOBAL_ALLOCATOR: RteAllocator = RteAllocator::new_uninitialized();
 
-fn init(args: impl IntoIterator<Item = impl AsRef<str>>) -> Eal {
```

Comment:
I think these changes to split up main are good, but I'll defer to @daniel-noland since he is more familiar with what has to happen for dpdk bring up.

Reply:
Daniel approved :tada: